operator/controller/src/main/java/io/apicurio/registry/operator/resource/app/AppDeploymentResource.java

```diff
@@ -74,7 +74,7 @@ protected Deployment desired(ApicurioRegistry3 primary, Context<ApicurioRegistry
                 .map(StorageSpec::getType).ifPresent(storageType -> {
                     switch (storageType) {
                         case POSTGRESQL -> PostgresSql.configureDatasource(primary, envVars);
-                        case KAFKASQL -> KafkaSql.configureKafkaSQL(primary, envVars);
+                        case KAFKASQL -> KafkaSql.configureKafkaSQL(primary, d, envVars);
```

Comment:
Can you rename `d` to `deployment` perhaps?  It's obvious to you, but in the GH diff I had to drill down to figure out what that was...